process/TI-Gives+Gets.md

```diff
@@ -0,0 +1,87 @@
+# "Gives and Gets" for OpenSSF Technical Initiatives (TI)
+
+The OpenSSF has a large community of contributors and efforts that span the broad spectrum of open source security interests.  The Technical Initiatives (TIs) of the foundation are where community members collaborate and help craft unique solutions to address improving the security of the open source ecosystem.
+In exchange for meeting certain requirements, the TIs are eligible to receive an assortment of benefits and have access to the capabilities of the Foundation's resources.  The specific requirements and benefits (aka "Gives and Gets") for each level of maturity are documented below.
+
+Based on the specific type of work the TI is focused on (e.g., software, specification, or documentation development) the requirements and benefits may slightly differ as applicable.
+
+Also note that benefits may actually vary based on resources and funds availability, or lack thereof.
+
+## Sandbox level Gives & Gets
+
+### Gives/Requirements
+
+ * TI must be aligned with the OpenSSF mission and either be a novel approach for existing areas or address an unfulfilled need. It is expected that the initial code needed for an OpenSSF WG to work be kept within their repository and will not function as a project in its own right. Should initial WG code grow and mature that it warrants its own Project status, then it is subject to Sandbox entry requirements. It is preferred that extensions of existing OpenSSF projects collaborate with the existing project rather than seek a new project.
+ * TI must maintain a diversified contributor base (i.e. not a single-vendor project).  TI must have a minimum of two maintainers with different organization affiliations.
+ * TI must find an aligned WG to host the TI and must have a TAC sponsor that can help guide the TI through processes.
+ * TI agrees to follow the [Secure Software Development Guiding Principles](https://github.com/ossf/wg-best-practices-os-developers/blob/main/docs/SecureSoftwareGuidingPrinciples.md) and the [Open Source Consumption Manifesto](https://github.com/ossf/wg-endusers/tree/main/MANIFESTO).
+ * If contributing an existing Project to the OpenSSF, the contribution must undergo license and IP due diligence by the Linux Foundation (LF).
+ * Provides quarterly updates to the TAC on technical vision and progress on vision.
+ * TI will have a [SECURITY.md](http://security.md/) that describes how the Project manages vulnerabilities, or more broadly how the OSSF handles vulnerability reports
+
+### Gets/Benefits
```

Comment:
There's definitely some overlap here in items and wording from the other files in `process/` (e.g. https://github.com/ossf/tac/blob/main/process/project-lifecycle.md). Are we okay with that duplication? Or do we want to focus on things that aren't already covered in lifecycle process documentation?

Reply:
Indeed. This document was started with what's in the project-lifecycle.md. The plan (that I suspected and CRob has now confirmed to me) is that this is meant to be a generalized version that will be applicable to all types of TIs. Once this is completed we'll remove the overlapping pieces from the project-lifecycle.md file and point to this file instead.